package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "piral-inspector",
-  "version": "0.11.0",
+  "version": "0.11.1",
```

Comment:
Great, IIRC we also need to change it in the manifest (https://github.com/smapiot/piral-inspector/blob/main/src/public/manifest.json#L11)